### PR TITLE
ISSUE-1733 Import common_clickhouse-srv

### PR DESCRIPTION
**Pull request by taiyang-li**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary
1. import clickhouse-srv as common package, related issue: https://github.com/datafuselabs/datafuse/issues/1733
2. Implement save for `DateTime64ColumnData` and `ChronoDateTimeColumnData`,  more details pls refer to https://github.com/sundy-li/clickhouse-srv/pull/6 

## Changelog
- Improvement

### CODE REVIEW

File: common/clickhouse-srv/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[package]
+name = "common-clickhouse-srv"
+version = "0.3.2"
+authors = ["sundyli <543950155@qq.com>"]
+edition = "2018"
```
Comment (PsiACE) [inline common/clickhouse-srv/Cargo.toml:5]: I think it should be migrated to the 2021 edition

File: common/clickhouse-srv/examples/simple.rs [hunk 2]
```diff
@@ -0,0 +1,182 @@
+use std::env;
```
Comment (PsiACE) [inline common/clickhouse-srv/examples/simple.rs:1]: add license header like other files

**Review by PsiACE — CHANGES_REQUESTED** [review]

File: common/clickhouse-srv/Cargo.toml (hunk 1 above)
Comment (PsiACE) [reply common/clickhouse-srv/Cargo.toml:5]: FYI https://blog.rust-lang.org/2021/07/21/Rust-2021-public-testing.html

Comment (taiyang-li) [reply common/clickhouse-srv/Cargo.toml:5]: ok !

File: common/clickhouse-srv/examples/simple.rs (hunk 2 above)
Comment (taiyang-li) [reply common/clickhouse-srv/examples/simple.rs:15]: Thanks. Done

File: common/clickhouse-srv/Cargo.toml (hunk 1 above)
Comment (taiyang-li) [reply common/clickhouse-srv/Cargo.toml]: Done.

**Review by sundy-li — APPROVED** [review]

**Pull request merged**